functions/utils/index.js

```diff
@@ -0,0 +1,116 @@
+import { RelationKind } from './constants';
+
+const isRecord = (value) =>
+  value &&
+  typeof value === 'object' &&
+  !Array.isArray(value) &&
+  Object.keys(value).length > 0 &&
```

Comment:
Is the check for an id not sufficient? Such as `Boolean(value?.id)`

Reply:
Not sure, but this is reused from the native repo so I assume it is necessary for some cases: https://github.com/bettyblocks/native-functions/blob/master/functions/utils/index.js.

Reply:
Every record should have an id right? So if this is correct then we can use this logic to determine if the record is present.